framework/src/st_things/things_stack/utils/things_network.c

```diff
@@ -259,31 +243,57 @@ int things_wifi_changed_call_func(int state, char *ap_name, char *ip_addr)
 
 static void *__attribute__((optimize("O0"))) t_things_wifi_join_loop(void *args)
 {
+	bool* is_wifi_retry_connect = (bool*) args;
+
+	if ((*is_wifi_retry_connect) == true) {
+		THINGS_LOG_E(TAG, "Retry to connect to home ap.");
+		things_network_connect_home_ap();
```

Comment:
Return checking needed, maybe.

Reply:
This function was used on thread(callback). and the return value never used. so we can ignore that.

Reply:
I mean return value of things_network_connect_home_ap